Tests/UnitTests/Reporters/JunitXMLReporterTest.jsl

```diff
@@ -47,7 +47,8 @@ setup_expr = Expr(
         // This allows us to write assertions that are reported as outside of any test.
         ::ut assert that(ut as expr(Name Expr(value)), matcher, label);
     );
-    report = Expr(Try(cached_report, cached_report = reporter << get report));
```

Comment:
This was added to address flaky tests in #79. Multiple evals of `report` would give different XML since the `id` is based on the current time. We had tests that depended on multiple evals of `report` always giving the same XML.

Reply:
When I use the cached_report, I would get errors since the report would not update even between tests. The ids are checking using the regex id="\d+_\d+" so that it does not matter what time it is when the report is generated.

Reply:
I see what you mean by the flaky test, sometimes the second match is a little past the originally matched time.

Reply:
Yes, see [JunitXMLReporterTest.jsl#L165-L169](https://github.com/sassoftware/jsl-hamcrest/blob/f91cfdac472b4c83635cb97fdb4b1a510082cd3a/Tests/UnitTests/Reporters/JunitXMLReporterTest.jsl#L165-L169). L165 and L169 assume that the report XML is exactly the same. But without caching, those two reports have the possibility to be different.

Maybe add a `Teardown` to the `test case` to delete `cached_report`. Though it should be removed anyway since each test clears symbols and uses a fresh test case IIRC. Unless you ran `setup_expr` outside of the `test case` as symbols that exist when the test starts are not cleaned up.

Reply:
I will explicitly delete ``cached_report`` before making the expression for report to ensure it is not set.

Reply:
done in d89763bf14213a03beced1c8efc34f90241f8d24